dyn_em/solve_em.F

```diff
@@ -183,8 +183,7 @@ SUBROUTINE solve_em ( grid , config_flags  &
    LOGICAL :: leapfrog
    INTEGER :: l,kte,kk
    LOGICAL :: f_flux  ! flag for computing averaged fluxes in cu_gd
-   REAL    :: curr_secs, curr_secs2, curr_mins2
-   REAL(8) :: curr_secs_r8, curr_secs2_r8
+   REAL    :: curr_secs, curr_secs2
```

Comment:
I don't think you should be making this change. The two lines in the code should be retained.